### PR TITLE
[Programs] Hacktoberfest: Add dropdown list for years past

### LINKED ISSUE #3204: [Programs] Hacktoberfest: Add dropdown list for years past
#### Current Behavior
The [Hacktoberfest page](https://layer5.io/programs/hacktoberfest) does not have a drop-down for users to visit participation details of previous years.

#### Desired Behavior
Akin to the year-over-year drop-down that you can find on the [Google summer of code](https://layer5.io/programs/gsoc) or other programs that we have participated in year over year, the Hacktoberfest page should have a similar drop-down.

#### Screenshots / Mockups
Screen Shot 2022-09-28 at 10 38 56 AM

#### Implementation
1. See GSoC page for an example.

#### Acceptance Tests
1. Visitors can navigate between different years of participation.

---
<h3> Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h3>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by XDRAGON2002**
Signed-off-by: Anant Vijay <anantvijay3@gmail.com>

**Description**
image

This PR fixes #3204 

**Notes for Reviewers**
Added a new directory for hacktober2021

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by XDRAGON2002** [comment]
I didn't have the graphic for hacktoberfest2021 or for the years before it, if someone could kindly share them I could add all the required years to the list as well.

**Comment by leecalcote** [comment]
2020 -

**Comment by leecalcote** [comment]
2021 -

**Comment by leecalcote** [comment]
You can look back in history for additional text describing those years, too. Here's one link, but there are other ways to browse history, too. https://github.com/layer5io/layer5/tree/250d2d8d301dff53fbd8f6d67830e8d6fe1fa3a1/src/collections/programs/hacktoberfest-2020

File: src/collections/programs/hacktoberfest-2021/index.mdx [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+<br/>
+
+<div>
+    While small, the Layer5 community represents the largest collection of
+      service mesh projects and their maintainers in the world. We build
+      projects to provide learning environments, deployment and operational best
+      practices, performance benchmarks, create documentation, share networking
+      opportunities, and more. Our shared commitment to the open source spirit
+      pushes Layer5 projects forward. New members are always welcome.
+</div><br/>
```
Comment (leecalcote) [inline src/collections/programs/hacktoberfest-2021/index.mdx:36]: @XDRAGON2002 please resync with `master` as the formatting and text of the 2022 page has been overhauled.

**Review by leecalcote — CHANGES_REQUESTED** [review]
Thanks, @XDRAGON2002 !

**Comment by l5io** [comment]
🚀 Preview for commit 4fa521ba33f237cc3f71fa9e5700c07ede9d8286 at: https://6335dd0ee7abb0238139fd5f--layer5.netlify.app

**Comment by XDRAGON2002** [comment]
@leecalcote synced with master, updated the graphics for previous years and also added page for 2020

**Comment by l5io** [comment]
🚀 Preview for commit e9a40f5234ddd75f9a93d3cf4f3c07f8e703eb6b at: https://6335ec16b6fd7b277c47d5e5--layer5.netlify.app

**Review by leecalcote — APPROVED** [review]
Thank you, @XDRAGON2002. Were you able to find any text specific to each year, by chance?

We need to get the Meshery Docker Extension listed in the 2022 page quickly.

**Comment by XDRAGON2002** [comment]
@leecalcote I went through the past year texts but found them to be more or less the same and hence didn't modify them and added them based on this years format, moreover just opened the PR for Meshery Docker Extension link as well, by listing it on the 2022 page are you referring to adding it to the hacktoberfest page?

**Comment by leecalcote** [comment]
@XDRAGON2002, thank you for looking. It sounds like we didn't have much by way of differentiated content from year to year other than the graphic thus far. Let's change that this year with mention of the Docker Meshery Extension.

**Comment by XDRAGON2002** [comment]
@leecalcote sure, I'll open another PR for that then, could you clarify a bit as to what the means of "mentioning" might look like? A dedicated button? Or maybe another link to dockerhub similar to this PR? Or maybe a link to the meshery docker-extension page?

**Pull request merged**